bedrock/careers/templates/careers/base.html

```diff
@@ -37,6 +37,11 @@
     'href': url('careers.home'),
     'cta_name': "Careers overview"
   },
+  {
+    'text': "Diversity and Inclusion",
+    'href': url('careers.diversity'),
+    'cta_name': "Mozilla Diversity and inclusion"
```

Comment:
**Nit**: Inconsistent capitalization here

Reply:
The pattern of titles on our navs usually have each word capitalized, so that's why I capitalized it

Reply:
oh, i'm dumb - I was looking at the cta-name not the text. Can forget about this